source/_integrations/zwave_js.markdown

```diff
@@ -149,11 +153,35 @@ These are events fired by the device using the notification command class. The `
     "node_id": 1,
     "home_id": "974823419",
     "device_id": "ad8098fe80980974",
-    "label": "Keypad lock operation",
+    "command_class": 113,
+    "command_class_name": "Notification",
+    "type_": 6,
```

Comment:
This is not current I think.

Reply:
They are as far as I can tell. The command class parameters tell you whether it's coming from `Notification` or `Entry Control` and the attributes match what we get from the event

Reply:
sorry, now I see what you are saying. Additional commit incoming

Reply:
```suggestion
    "type": 6,
```